### ISSUE TITLE
PDF Template should have a qr code generated based on parameters

### ISSUE THREAD

**Issue by MWernerS** [COSC481W-2024Winter/augmentedDocScanner#49]
User Story: The user wants to get a template that is a different size than normal.

- Persona: A user wants a custom template
- Feature: The template currently has an unchanging qr code. We want the qr code to have encoded onto it the information of the page it is printed on.
- Business value: Wider range of purpose for product

Tasks
- [ ] A default if nothing in fields
- [ ] A way to obtain a qr code with the desired information
- [ ] New qr code replaces old qr code on template

Acceptance criteria:
- [ ] QR code on template changes based on what the user types into the website

Depends on: Issue #38

**Comment by MWernerS** [comment]
Duplicate of #38 . Requirements of #38 include this.
